_includes/docs-info.html

```diff
@@ -42,17 +42,19 @@ <h6>For Users</h6>
         <a name="tutorials"></a>
         <h5>Tutorials for the finer steps</h5>
             <div class="row docs-details">
-                <a href="https://bit.ly/2Fyn97v">Spherical Harmonic Representation Point Distribution Models (SPHARM-PDM)</a> Last updated July 21st 2017.
+                <a href="https://bit.ly/3q73CTo">Spherical Harmonic Representation Point Distribution Models (SPHARM-PDM)</a> Last updated September 2022.
                 <br/>
-                <a href="https://bit.ly/2WsFiun">Study-specific shape analysis (GROUPS)</a> Last updated July 16th 2018.
+                <a href="https://bit.ly/3q6v1oo">Study-specific shape analysis (GROUPS)</a> Last updated September 2022.
                 <br/>
-                <a href="http://bit.ly/2uVYche">Shape Regression</a> Last updated February 5th 2019.
+                <a href="https://bit.ly/3KKLWGO">Shape Regression</a> Last updated September 2022.
                 <br/>
-                <a href="https://bit.ly/2HYbHVA">Shape Variation Analyzer</a> Last updated December 9th 2019.
+                <a href="https://bit.ly/3cHG704">Shape Variation Analyzer</a> Last updated September 2022.
                 <br/>
-                <a href="http://bit.ly/30MiJ6v">Shape analysis via skeletal models</a> Last updated July 1st 2019.
+                <a href="http://bit.ly/30MiJ6v">Shape analysis via skeletal models</a> Last updated September 2022.
                 <br/>
-                <a href="http://bit.ly/319KVBN">Covariate Significance Testing</a> Last updated July 28th 2020.
+                <a href="http://bit.ly/319KVBN">Covariate Significance Testing</a> Last updated September 2022.
+                <br/>
+                <a href="https://bit.ly/3AGp2eR">Surface model segmentation (DentalModelSeg) </a> Last updated September 2022.
```

Comment:
https://docs.google.com/presentation/d/1YasWQvgINKa_nCPGGBwvsAEU6dfFxl-H4qaYekpQtZI/present?slide=id.p1

It is a little weird the Kitware logo has a white background here, but not in any of the other tutorials

Reply:
https://docs.google.com/presentation/d/1YasWQvgINKa_nCPGGBwvsAEU6dfFxl-H4qaYekpQtZI/present?slide=id.p6
https://docs.google.com/presentation/d/1YasWQvgINKa_nCPGGBwvsAEU6dfFxl-H4qaYekpQtZI/present?slide=id.p7

I can't click on the hyperlinks.

Reply:
Should probably add a final slide with the github repo links and a contact, similar to the other tutorials.

Reply:
Kitware logo is fixed.
Hyperlinks are fixed.
Final slide added.